framework/base/BaseObject.php

```diff
@@ -150,6 +149,8 @@ public function __set($name, $value)
             $this->$setter($value);
         } elseif (method_exists($this, 'get' . $name)) {
```

Comment:
Is it expected that existing getter will prevent property setting?

Reply:
If someone only writes a getter and ommits a setter I'd say it's a clear decision make by the developer. So in that case the error about writing a read-only property should still be thrown.

Reply:
Yes, and it's BC safe.